context.go

```diff
@@ -639,3 +669,18 @@ func (c *context) resolveXAttrs(fp string, fi os.FileInfo, base *resource) (map[
 
 	return nil, nil
 }
+
+// resolveTimestamps set base.[AMCB]Times.
+// When BTime is unsupported, it is set to the Unix epoch.
+func (c *context) resolveTimestamps(fi os.FileInfo, base *resource) {
+	ts := times.Get(fi)
+	base.atime = ts.AccessTime()
+	base.mtime = ts.ModTime()
+	base.ctime = ts.ChangeTime()
+	if ts.HasBirthTime() {
+		base.btime = ts.BirthTime()
+	} else {
+		// Note: time.Time{} stands for 0001-01-01 00:00:00 UTC
+		base.btime = time.Unix(0, 0)
```

Comment:
I'm very worried about the redefinition of the zero time. This should really be `time.Time{}`.